rcl_yaml_param_parser/src/parse.c

```diff
@@ -18,14 +18,54 @@
 #include <string.h>
 
 #include "rcutils/allocator.h"
+#include "rcutils/format_string.h"
 #include "rcutils/strdup.h"
 
+#include "rmw/error_handling.h"
+#include "rmw/validate_namespace.h"
+#include "rmw/validate_node_name.h"
+
 #include "./impl/add_to_arrays.h"
 #include "./impl/parse.h"
 #include "./impl/namespace.h"
 #include "./impl/node_params.h"
 #include "rcl_yaml_param_parser/parser.h"
 
+///
+/// Check a name space whether it is valid
+///
+/// \param[in] namespace the namespace to check
+/// \return RCUTILS_RET_OK if namespace is valid, or
+/// \return RCUTILS_RET_INVALID_ARGUMENT if namespace is not valid, or
+/// \return RCUTILS_RET_ERROR if an unspecified error occurred.
+RCL_YAML_PARAM_PARSER_LOCAL
```

Comment:
We should include `rcl_yaml_param_parser/visibility_control.h`.

Reply:
OK. I'll add it.
(Actually, I thought rcl_yaml_param_parser/parser.h already includes rcl_yaml_param_parser/visibility_control.h, so I ignore it before.)

Reply:
Maybe it's redundant, but I think it's generally good to directly include whatever is being used in a file.